lib/ansible/modules/packaging/language/composer.py

```diff
@@ -221,7 +239,9 @@ def main():
         'no_dev': 'no-dev',
         'no_scripts': 'no-scripts',
         'no_plugins': 'no_plugins',
```

Comment:
not related to your contribution but is it just me or shouldn't this look like

~~~diff
-         'no_plugins': 'no_plugins',
+         'no_plugins': 'no-plugins',
~~~

Reply:
Indeed. I can fix it if you want ?

Reply:
feel free